@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -421,67 +480,249 @@ export function getMaterialConstructor(technique: Technique): MaterialConstructo
+ *
+ * @param technique an technique the applied color comes from
+ * @param material the material to which color is applied
+ * @param prop technique property (color) name
+ * @param value color value
+ * @param level optional, tile zoom level for zoom-level dependent properties are evaluated.
+ */
+function applyTechniqueColorToMaterial(
+    technique: Technique,
+    material: THREE.Material,
+    prop: string | number,
+    value: any,
+    level?: number
+) {
+    const m = material as any;
+    assert(m[prop] instanceof THREE.Color);
+    assert(
+        !isBaseColorProp(technique, prop),
+        "Main (transparent) technique colors should not be processed here!"
+    );
+
+    value = evaluateColorProperty(value, level);
+
+    if (ColorUtils.hasAlphaInHex(value)) {
+        logger.warn("Used RGBA value for technique color without transparency support!");
+        // Just for clarity remove transparency component, even if that would be ignored
+        // by THREE.Color.setHex() function.
+        value = ColorUtils.removeAlphaFromHex(value);
+    }
+
+    m[prop].setHex(value);
+    // Trigger setter notifying change
+    m[prop] = m[prop];
+}
+
+/**
+ * Apply technique base color (transparency support) to material with modifying material opacity.
+ *
+ * This method applies main (or base) technique color with transparency support to the corresponding
+ * material color, with an effect on entire [[THREE.Material]] __opacity__ and __transparent__
+ * attributes.
+ *
+ * @note Transparent colors should be processed as the very last technique attributes,
+ * since their effect on material properties like [[THREE.Material.opacity]] and
+ * [[THREE.Material.transparent]] could be overridden by corresponding technique params.
+ *
+ * @param technique an technique the applied color comes from
+ * @param material the material to which color is applied
+ * @param prop technique property (color) name
+ * @param value color value in custom number format
+ * @param level optional, tile zoom level for zoom-level dependent properties are evaluated.
+ */
+function applyTechniqueBaseColorToMaterial(
+    technique: Technique,
+    material: THREE.Material,
+    prop: string | number,
+    value: any,
+    level?: number
+) {
+    const m = material as any;
+    assert(m[prop] instanceof THREE.Color);
+    assert(
+        isBaseColorProp(technique, prop),
+        "Secondary technique colors should not be processed here!"
+    );
+
+    value = evaluateColorProperty(value, level);
+
+    const { r, g, b, a } = ColorUtils.getRgbaFromHex(value);
+    // Override material opacity and transparency by mixing technique defined opacity
+    // with main color transparency
+    const tech = technique as any;
+    let opacity = a;
+    if (tech.opacity !== undefined) {
+        opacity *= evaluateProperty(tech.opacity, level);
+    }
+    opacity = THREE.Math.clamp(opacity, 0, 1);
+    let transparent = opacity !== 1.0;
```

Comment:
As discussed this has the side effect of changing the render order. (despite of the fact that there are still artifacts that @robertoraggi is looking into). I would rather not touch the transparent flag but just make sure that blending is enabled.

Reply:
Done, we're using `enable/disableBlending` that is compatible with #1144.